src/components/interaction-designer/block-types/MobilePrimitives_NumericResponseBlock.vue

```diff
@@ -24,6 +24,10 @@
         :has-ivr="hasVoiceMode"
         @commitMaxDigitsChange="updateMaxDigits" />
 
+      <block-output-branching-config
```

Comment:
Missing `<hr>` before `<block-output-branching-config>`
(according to design: https://www.figma.com/proto/U2UHUAFgkM0sBdBiinh6ah/DSN-110)

Reply:
For all 03 blocks I think, maybe we just need to insert `<hr>` inside `block-output-branching-config` component directly ?

Reply:
I'd prefer to see the `<hr>`s at the block vue file level, it gives most flexibility!